.github/workflows/mkosi.yml

```diff
@@ -73,7 +73,7 @@ jobs:
 
     steps:
     - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c
-    - uses: systemd/mkosi@268a9374d5a93a4021db6521a39db2c2ed9fd755
+    - uses: systemd/mkosi@f642822e59a274697544e9d2cef8c6539e6b5ae2
```

Comment:
Hmm, is f642… a commit in mkosi git?

Reply:
I think Daan used the ref from the PR https://github.com/systemd/mkosi/pull/1454 which got changed, since it got rebased on merge.

Reply:
Wow. Just wow. So you can use a git hash that is resolvable because of github shenanigans, but it doesn't even need to be part of the repo. Lovely.

Reply:
It is quite convenient for testing I must say